style/js/screen_gfx.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+export const spriteUpscale = window.devicePixelRatio > 1 ? 1 : 2;
+
+export const fontSizeType = "px";
```

Comment:
Suggestion to remove duplication in producing the font specification, and to select a narrow font for improved fit (sans-serif-condensed maps to Roboto Condensed on Android, and Arial Narrow is available on Chrome OS. Unsure about others.)

```suggestion
const fontFamily = '"sans-serif-condensed", "Arial Narrow", sans-serif';
export const shieldFont = size => `bold ${size}px ${fontFamily}`;
```

Then replace the four font strings with a call to this function.

Reply:
e.g. Roboto:

![Screenshot_20220115-092803](https://user-images.githubusercontent.com/23022/149625659-525aaa2a-15c9-4bbf-ad97-823e5acc5563.png)